index.css

```diff
@@ -96,3 +102,78 @@ main {
     margin: auto;
     text-align: center;
 }
+
+
+.wrapper {
+    margin: 0 auto;
+    background-color: #ffffff;
+    height: 50px;
+    padding: 0;
+
+
+
+}
+
+
+header {
+    display: flex;
+    justify-content: space-between;
+    padding: 5px;
+    border-radius: 0;
+    position: fixed;
+    top: 0;
+    width: 100%;
+    background-color: #ffffff;
+
+
+}
+
+header img.logo {
+    padding: 5px;
+    width: 40px;
+    height: 40px;
+}
+
+header img.profile {
+    width: 40px;
+    height: 40px;
+    padding: 5px;
+    border-radius: 50%;
+
+
+}
+
+.search-container{
+    border: 2px;
+
+
+}
+.search-container input[type=text] {
+    padding: 2px;
+    margin-top: 5px;
+    font-size: 20px;
+    width: 400px;
+    height: 40px;
+    border: thin solid gainsboro;;
+
+
+}
+
+.search-container button {
+    margin-top: 5px;
+    font-size: 15px;
+    float: right;
+    border: none;
+    background: #4169e1;
+    color: #ffffff;
+    padding: 4px;
+    width: 100px;
+    height: 40px;
+    border-top-right-radius: 10%;
+    border-bottom-right-radius: 10%;
+
+
+
+}
```

Comment:
I would suggest these changes. Button takes automatically 20% of the search container width. Design is a bit more consistent with the rest of the content. Also added different background color when hovering over the search button.

.search-container button {
    width: 20%;
    height: 80px;
    padding: 4px;
    font-size: 28px;
    font-weight: 500;
    float: right;
    border: none;
    color: #ffffff;
    background: dodgerblue;
}

.search-container button:hover {
    background: rgb(0, 124, 248);
}